package.json

```diff
@@ -32,13 +32,13 @@
     "gulp": "^3.9.0",
     "gulp-eslint": "^0.14.0",
     "gulp-mocha": "^2.1.1",
-    "mocha": "^2.2.5",
-    "node-sass": "^3.3.3",
-    "should": "^6.0.3"
+    "mocha": "^2.3.3",
+    "should": "^7.1.0"
   },
   "dependencies": {
     "lwip": "^0.0.8",
-    "node-sass-utils": "^1.1.1",
+    "node-sass": "^3.4.2",
```

Comment:
Can you move `node-sass` back into `devDependencies`

This is something we're still working through on the module side, but we've seen weird issues in the past when a module and a build system both declare node-sass dependencies. It's better to just let the build system declare it's dependency.

We need it in `devDependencies` for testing.